examples/export-data/main.ts

```diff
@@ -1,2 +1,183 @@
-// Run this with yarn export-data or npx tsx main.ts
-console.log("Hello World")
\ No newline at end of file
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import { dbConnect } from './utils/db-connect';
+import { AnalyticsViewer, AnalyticsLogger, AnalyticsManager } from "bog-analytics";
+import { ClickEventModel } from './models/click-event';
+import { VisitEventModel } from './models/visit-event';
+import { InputEventModel } from './models/input-event';
+import { CustomEventModel } from './models/custom-event';
+import { Project } from './utils/types';
+import { EventEnvironment } from './utils/types';
+import { getProjectIDByName, createProject } from './actions/project'
+
+dotenv.config();
+const CLIENT_KEY = process.env.CLIENT_API_SECRET || '';
+const SERVER_KEY = process.env.SERVER_API_SECRET || '';
```

Comment:
I would rename this to `process.env.CLIENT_API_KEY` and `process.env.SERVER_API_KEY` since those a tad bit more descriptive

can you make an .env.example file for people to reference - should be pretty simple; something like:

```
CLIENT_API_KEY=""
SERVER_API_KEY=""
DATABASE_URL=""
DATABASE_NAME=""
```